tools/testsys/Test.toml.example

```diff
@@ -4,15 +4,38 @@
 # This section contains configuration details for all testing
 [test]
 
+# Configurable values:
```

Comment:
These shouldn't be in the `[test]` table since they cannot be added there. I still prefer them at the bottom.

Reply:
When I was first going through the example TOML file, I went from top to bottom so I had a hard time grokking what each field was suppose to do and went into the code to look for answers until I finally reached the bottom. I think if it's not for `[test]` can we at least move it below "Test Configuration"?

Reply:
That sounds good.